multimodal/src/autogluon/multimodal/utils/inference.py

```diff
@@ -32,13 +32,13 @@ def extract_from_output(outputs: List[Dict], ret_type: str, as_ndarray: Optional
     """
     if ret_type == LOGITS:
         logits = [ele[LOGITS] for ele in outputs]
-        ret = torch.cat(logits)
+        ret = torch.cat(logits).nan_to_num(nan=-1e4)
```

Comment:
Does nan often appear in the ft transformer outputs? Is the nan in inference normal? Nans in training can cause model collapse.

Reply:
I chatted with @sxjscience about this. I tested on 104 datasets and the Nan seems to only appear during validation in my experiments.